ui/src/hooks/useFileManage.ts

```diff
@@ -318,6 +415,23 @@ export const useFileManage = () => {
     if (fileConnectionUrl) {
       const socket = fileSocketConnection(fileConnectionUrl, message);
 
+      mittBus.on(
+        'file-upload',
+        ({
+          fileList,
+          onFinish,
+          onError,
+          onProgress
+        }: {
+          fileList: Ref<Array<UploadFileInfo>>;
+          onFinish: () => void;
+          onError: () => void;
+          onProgress: (e: { percent: number }) => void;
+        }) => {
+          handleFileUpload(<WebSocket>socket, fileList, onProgress, onFinish, onError);
+        }
+      );
+
       mittBus.on('download-file', ({ path, is_dir }: { path: string; is_dir: boolean }) => {
         handleFileDownload(<WebSocket>socket, path, is_dir);
       });
```

Comment:
The code does not contain any known irregularities, potential issues, or optimizations. It appears to be well-designed and follows common practices when working with Vue Router, `naive-ui`, WebSocket communication, and Vuex store. If there were such concerns identified at the time these changes were implemented or during development, they may have been logged rather than being part of the current codebase itself.

The focus should remain on understanding the current functionality without delving into historical context, as you requested in your request to respond using English only.